### ISSUE TITLE
crash in Android while running fastspeech 

### ISSUE THREAD

**Issue by manmay-nakhashi** [TensorSpeech/TensorflowTTS#101]
Caused by: java.lang.IllegalArgumentException: DataType error: DataType 6 is not recognized in Java (version 2.4.0)
        at org.tensorflow.lite.DataType.fromC(DataType.java:79)
        at org.tensorflow.lite.Tensor.<init>(Tensor.java:484)
        at org.tensorflow.lite.Tensor.fromIndex(Tensor.java:44)
        at org.tensorflow.lite.NativeInterpreterWrapper.getInputTensor(NativeInterpreterWrapper.java:303)
        at org.tensorflow.lite.NativeInterpreterWrapper.run(NativeInterpreterWrapper.java:140)
        at org.tensorflow.lite.Interpreter.runForMultipleInputsOutputs(Interpreter.java:343)

it's for attention mask
@dathudeptrai did you face this issue ??

**Comment by dathudeptrai** [comment]
@manmay-nakhashi can you try to ignore attention mask, you can calculate it by tf.math.not_equal(input_ids, 0) inside a function.

**Comment by manmay-nakhashi** [comment]
@dathudeptrai sure i can try that

**Comment by manmay-nakhashi** [comment]
@dathudeptrai this resolved my issue thanks : )
